cuBERT_topic_modelling/benchmarks/benchmark_cuBERTopic.ipynb

```diff
@@ -0,0 +1,172 @@
+{
```

Comment:
We need to fix the Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false) .
You can set environment variables using something like below:
```
import os
os.environ["TOKENIZERS_PARALLELISM"] = "true"
```